app/routes.py

```diff
@@ -1,2 +1,291 @@
+    request_body = request.get_json()
+    if video == None:
+        return jsonify({"details": "video not found"}), 404
+
+    else:
+        errors = ""
+        information = ["title", "release_date", "total_inventory"]
+        for info in information:
+            if info not in request_body:
+                errors = errors + info 
+                if errors != "":
+                    return jsonify({"details": errors}), 400
+
+    video.title = request_body["title"]
+    video.release_date = request_body["release_date"]
+    video.total_inventory = request_body["total_inventory"]
+
+    db.session.commit()
+    return jsonify({
+        "id": video.id,
+        "title": video.title,
+        "release_date": video.release_date,
+        "total_inventory": video.total_inventory
+    }), 200
+
+@video_bp.route("/<id>", methods = ["DELETE"])
+def delete_video(id):
+    video = Video.query.get(id)
+    if video == None:
+        return jsonify({"details": "video not found"}), 404
+
+    else: 
+        db.session.delete(video)
+        db.session.commit()
+        return jsonify({"id": video.id})
+
+# -----------------------------RENTAL ENDPOINTS-----------------------------
+
+@rental_bp.route("/check-out", methods = ["POST"])
+def rental_check_out():
+    request_body = request.get_json()
+
+    customer_id = request_body.get("customer_id")
+    video_id = request_body.get("video_id")
+    
+    if type(customer_id) != int or type(video_id) != int:
+        return jsonify({"details": "Invalid data"}), 400
+
+    customer = Customer.query.get(customer_id)
+    video = Video.query.get(video_id)
+
+    if customer is None and video is None:
+        return jsonify({"details": "id's do not exist"}), 404
+
+    if video.available_inventory < 1:
+        return jsonify({"details": "2Invalid data"}), 400
+
+
+    customer.videos_checked_out_count += 1
+    video.available_inventory -= 1
+
+    new_rental = Rental(
+        customer_id = customer_id,
+        video_id = video_id,
+        due_date = datetime.now() + timedelta(days = 7)
+    )
+
+    db.session.add(new_rental)
+    db.session.commit()
+
+    return jsonify({
+        "customer_id": new_rental.customer_id,
+        "video_id": new_rental.video_id,
+        "due_date": new_rental.due_date,
+        "videos_checked_out_count": customer.videos_checked_out_count,
+        "available_inventory": video.available_inventory
+    }), 200
+
+
+@rental_bp.route("/check-in", methods = ["POST"])
+def rental_check_in():
+    request_body = request.get_json()
+
+    customer_id = request_body.get("customer_id")
+    video_id = request_body.get("video_id")
+    
+    customer = Customer.query.get(customer_id)
+    video = Video.query.get(video_id)
+
+    for rental in customer.video:
+        if rental.video_id == video_id:
+            customer.videos_checked_out_count -= 1
+            video.available_inventory += 1
+            db.session.delete(rental)
+            db.session.commit()
```

Comment:
This will work, but it is also possible to pull the rental from the database directly.  This query ```Rental.query.filter_by(customer_id = customer_id, video_id=video_id).all()``` is a place to start.